### PR TITLE
Feature/as 276  end to end test for scheduled airing

### PR DESCRIPTION
**Pull request by ping2ravi**

### CODE REVIEW

File: src/test/resources/json/VQDataWithDeleteEntities.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (navindesai) [inline src/test/resources/json/VQDataWithDeleteEntities.json]: do we need VQ in EE. Maybe just the naming is wrong.

File: src/test/resources/json/VQTitle.json [hunk 2]
```diff
@@ -0,0 +1,353 @@
+{
```
Comment (navindesai) [inline src/test/resources/json/VQTitle.json]: VQTile is not present in EE.

File: src/test/java/com/sky/search/ee/stepdefs/util/JsonDataBuilder.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.spi.json.JacksonJsonNodeJsonProvider;
+import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import org.apache.commons.io.IOUtils;
+
+import java.util.Map;
+
+public class JsonDataBuilder {
+
+    public static String buildVQDataMessageFromParams(Map<String, String> params, String fileNameAndPath) throws Exception {
```
Comment (navindesai) [inline src/test/java/com/sky/search/ee/stepdefs/util/JsonDataBuilder.java]: dont need VQ data.... or maybe just the naming is incorrect

File: README.md [hunk 4]
```diff
@@ -1 +1,73 @@
...
+README: search-ee-e2e
+--------------------
+This module provides end-to-end tests that assume that the ethan europe search stack
+under test is already up and running.  The tests do NOT start an environment
+to run the tests against.
+
+To start up the stack, run `./gradlew setupE2e`
+
+The following properties configure the test environment:
+mongo.name
```
Comment (ping2ravi) [inline README.md]: Done

File: docker/deploy-consumer-topology.sh [hunk 5]
```diff
@@ -0,0 +1,22 @@
+docker run -e "topologyMode=DELTA" \
```
Comment (ping2ravi) [inline docker/deploy-consumer-topology.sh]: Deleted file

File: src/main/java/com/sky/search/e20m/ZookeeperConfig.java [hunk 6]
```diff
@@ -0,0 +1,73 @@
+package com.sky.search.e20m;
+
+import com.google.common.collect.Lists;
+
+public class ZookeeperConfig {
```
Comment (ping2ravi) [inline src/main/java/com/sky/search/e20m/ZookeeperConfig.java]: Removed files related to zookeeper

File: src/test/java/com/sky/search/ee/spring/ApiConfig.java [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    public String getSuggestApiHostUrl() {
+        return suggestApiHostUrl;
+    }
+
+    public String getReconcileApiHostUrl() {
+        return reconcileApiHostUrl;
+    }
+//    public static void main(String[] args) {
```
Comment (ping2ravi) [inline src/test/java/com/sky/search/ee/spring/ApiConfig.java]: Leaving It for now

File: src/test/java/com/sky/search/ee/stepdefs/BaseStepDef.java [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@ContextConfiguration(classes = Config.class)
+public class BaseStepDef {
+
+    protected static final String UK_TENANT = "UK";
```
Comment (ping2ravi) [inline src/test/java/com/sky/search/ee/stepdefs/BaseStepDef.java]: Done

File: src/test/java/com/sky/search/ee/stepdefs/BaseStepDef.java [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    protected static final String SOLR_HOST = "localhost:8983";
+    protected static final String SEARCH_URI = "http://localhost:4000";
```
Comment (ping2ravi) [inline src/test/java/com/sky/search/ee/stepdefs/BaseStepDef.java]: Deleted

File: src/test/java/com/sky/search/ee/stepdefs/util/JsonDataBuilder.java (hunk 3 above)
Comment (ping2ravi) [inline src/test/java/com/sky/search/ee/stepdefs/util/JsonDataBuilder.java]: Will be taken care in next Scheduled airing card

**Pull request merged**